docs/helm-chart-checks.md

```diff
@@ -124,59 +124,77 @@ This section provides help on the basic usage of Helm chart checks with the podm
 - Run a subset of the checks:
 
   ```
-  $ podman run --rm -i \
-          -e KUBECONFIG=/.kube/config \
-          -v "${HOME}/.kube":/.kube \
+  $ podman run --rm -i                                  \
+          -e KUBECONFIG=/.kube/config                   \
+          -v "${HOME}/.kube":/.kube                     \
           "quay.io/redhat-certification/chart-verifier" \
-          verify -e images-are-certified,helm-lint \
+          verify -e images-are-certified,helm-lint      \
           <chart-uri>
-  
+
   ```
 - Run all the checks except a subset:
 
   ```
-  $ podman run --rm -i \
-          -e KUBECONFIG=/.kube/config \
-          -v "${HOME}/.kube":/.kube \
+  $ podman run --rm -i                                  \
+          -e KUBECONFIG=/.kube/config                   \
+          -v "${HOME}/.kube":/.kube                     \
           "quay.io/redhat-certification/chart-verifier" \
-          verify -x images-are-certified,helm-lint \
+          verify -x images-are-certified,helm-lint      \
           <chart-uri>
     ```
 - Provide chart-override values:
 
   ```
-  $ podman run --rm -i \
-          -e KUBECONFIG=/.kube/config \
-          -v "${HOME}/.kube":/.kube \
+  $ podman run --rm -i                                  \
+          -e KUBECONFIG=/.kube/config                   \
+          -v "${HOME}/.kube":/.kube                     \
           "quay.io/redhat-certification/chart-verifier" \
-          verify -S default.port=8080 \
+          verify -S default.port=8080                   \
           <chart-uri>
   ```
 - Provide chart-override values from a file in the current directory:
 
   ```
-  $ podman run --rm -i \
-          -e KUBECONFIG=/.kube/config \
-          -v "${HOME}/.kube":/.kube \
-          -v $(pwd):/values \  
+  $ podman run --rm -i                                  \
+          -e KUBECONFIG=/.kube/config                   \
+          -v "${HOME}/.kube":/.kube                     \
+          -v $(pwd):/values                             \
           "quay.io/redhat-certification/chart-verifier" \
-          verify -F /values/overrides.yaml \
+          verify -F /values/overrides.yaml              \
           <chart-uri>
-
   ```
 
+## Using the `chart-verifier` binary for Helm chart checks (Linux only)
+
+Alternatively, download `chart-verifier` binary from the [release page](https://github.com/redhat-certification/chart-verifier/releases) and run `chart-verifier verify` command to perform Helm chart checks, as explained in [Procedure](#procedure) section. For example,
+
+```
+$ podman run --rm -i                                  \
+        -e KUBECONFIG=/.kube/config                   \
+        -v "${HOME}/.kube":/.kube                     \
+        "quay.io/redhat-certification/chart-verifier" \
+        verify                                        \
+        <chart-uri>
+```
+
+will become (assuming the binary is downloaded and unzipped under current working directory)
```

Comment:
This also assume kube config is under $HOME/.kube. Otherwise set environment variable KUBECONFIG

Reply:
Updated, nice spot.